### ISSUE TITLE
Request for any environment results in GPU server (and spawn failure)

### ISSUE THREAD

**Issue by cullen-molitor** [microsoft/PlanetaryComputer#63]
I have been unable to get access to a `GPU PyTorch` server for the last week. I thought I would try my luck on the weekend assuming there would be fewer GPU users. I was again unable to launch a server so I thought I would try a normal python environment. The message is either stuck on GPU or it is trying to actually launch a GPU server. Either way, I am unable to access any server, and I think it is unlikely my home directory is full. 

Here is a screenshot from my attempt to launch a `CPU python` environment. 

![image](https://user-images.githubusercontent.com/56735240/169662747-a5002db9-5235-4ad6-b25a-47e85fb93623.png)

**Comment by TomAugspurger** [comment]
Mmm, the spawn failing on the CPU profile is surprising. Let me take a look.

**Comment by cullen-molitor** [comment]
I was able to access a `GPU PyTorch` server over the weekend and yesterday. This morning I was unable, and I thought I would test other options again. I first tried an `R` server with the same result as above (a message about insufficient GPU and spawn failure after 900 seconds). I then tried `CPU python` with the same result as above except the final message was a bit simpler `Spawn failed: Timeout`. I then tried `QGIS` with the same result. The common theme is the message about `insufficient nvidia.com/gpu`.
